src/compiler/diagnosticMessages.json

```diff
@@ -1923,6 +1923,10 @@
         "category": "Error",
         "code": 2685
     },
+    "Identifier '{0}' must be imported from a module": {

```

Comment:
But that isn't specifically the solution. The problem is that you're using something intended to be used as a global. Importing an identifier by that name from a module might not work at all. Importing the module *as* that name would work.

A better message might be

```
The '{0}' namespace declared in '{0}' is not accessible within a module. Did you mean to import the file instead?
```